lib/round.rb

```diff
@@ -0,0 +1,61 @@
+class Round
+  attr_reader :deck,
+              :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards.first
+  end
+
+  def take_turn(guess)
+    @turns << Turn.new(guess, current_card)
+    @deck.cards.rotate!(1)
+    @turns.last
+  end
+
+  def number_correct
+    @turns.count{ |turn| turn.correct?}
```

Comment:
This is great! Another way you can write this is:
`@turns.count(&:correct?)`

Neither is better than the other, but this class is already very well written and refactored, I thought you might like to see some other ruby tricks.